net-analyzer/gvm-libs/Manifest

```diff
@@ -1 +1,5 @@
 DIST gvm-libs-21.4.4.tar.gz 298551 BLAKE2B 271716d8be2376e2b54218b95069e19d10939c5feda9894220418bf184c4098eeacf61d1faedd49fe3215aa0fbf4a4afcfe2a929ac88450502737eb8af8f8e35 SHA512 6c80820c830cbc2e62865a0823740d857db075f2faa1a1e58af7a8f07e0179e7a78934d7613ec57f144900a5530206d4c07c912a6bbe40ad9d0549f9443bfb12
+DIST gvm-libs-22.4.6.tar.gz 311092 BLAKE2B 852f0787ccb86e4bc3a219de3567608001c25b4cddd7729e497715068581f98f20c18536027a344d38f32647dca89882d56fb397f8f60a37ff119f1e69bd2913 SHA512 c5ae33f77a4e44d2b383752725854ef7b1ffcea046361fb1400d2c30c315b39dc4f9059a31b0729e92b158ed42416dbff554143a12bde9422b84f48ea609775d
+DIST gvm-libs-22.5.2.tar.gz 312449 BLAKE2B 68a86ed8109c95f6c7a30712205e54463ed69379c6210140519a1f73df56582491d1c38b5907dffa10a9edcc59729a172c1b4052f6b5613a29045e51092c8ab9 SHA512 5bf3dd6871a72ceb8c59763ae5877a7cfe3fe14be1410eb00a78859e38dc42d536985cf78446aad115e8995d2f9ff2b29191038fc8cb4943e3c6e999ea856b63
+DIST gvm-libs-22.6.0.tar.gz 313140 BLAKE2B 79d807237aff12998167144c4671d0295ea4d550d21cc3b4716203ba6fbc1f285e77e23ad5f5c0890c93a3cca4abb20456b825cf6bb541044c6e4fe4139bbf8a SHA512 c8f5329040e0363ad592b54b256ce6025b18869f5d67cd8b21c68f09d04dafec46a71a753c2a5d9eeb815706c25af3c02e15b79aa7541a645ca8e945aa18fee9
+DIST gvm-libs-22.6.1.tar.gz 314446 BLAKE2B 375bf8567f1db597f278e6ff0646be52cb55ad7b93e5ab27e98bec6d89f25bf4c3c9b94925b5ed3d2c08fe6488e26af0e14a69fab5731ef7ce1543b2bd79f2e0 SHA512 73201129d6ea593415d5f33af526e39879e3f81280e91c4f17bc1add82c277bd89f4b7187ac53d06bcba8e230c8a159fcf02fe4dcd684c54675e7406ea4e6680
```

Comment:
Why do we need to add four versions of gvm-libs? Would just adding 22.6.1 enough? Or maybe 22.6.1 and 22.5.2? Or maybe 22.6.1 and 22.4.6?

Reply:
Reading here: https://forum.greenbone.net/t/version-compatibility/14689/2 all versions 22.x will be compatible with 22.4. 
Version 22.4.6 is the first one I added when I started writing the ebuild.
Then later versions were released which I tested and added.
I think we could just leave only version 22.6.1